cmd/root.go

```diff
@@ -0,0 +1,16 @@
+package cmd
+
+import (
+	"github.com/spf13/cobra"
+)
+
+func RootCmd() *cobra.Command {
+	var rootCmd = &cobra.Command{
+		Use:   "eoe",
+		Short: "EigenLayer On-chain Exporter (eoe) exposes Prometheus metrics about EigenLayer's Node Operator.",
```

Comment:
Is this right? I thought it was exposing metrics about EigenLayer's AVSs onchain events.

Reply:
Both AVS and operator's events.

Reply:
Fixed https://github.com/NethermindEth/eigenlayer-onchain-exporter/pull/3/commits/42ee4ebf92a370b122ad80ae69f17f8346917afb